.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
Should we enable `package-lock` to leverage `npm audit fix`?

Reply:
Sure, I don't mind. :) --done.